### PR TITLE
finally push the document shapes and cpu drawing

### PR DESCRIPTION
**Pull request by TeXhackse**
as discussed and sorry that i finished and forgot to push … again …

### CODE REVIEW

File: examples.sty [hunk 1]
```diff
@@ -148,9 +156,9 @@
 \cs_new:Nn
   \file:n
   {
-    \small_icon:n
-      { #1 }
-    \,
+%    \small_icon:n
+%      { #1 }
+%    \,
```
Comment (Witiko) [inline examples.sty:162]: Let's undo this change, as discussed with Karl.
```suggestion
    \small_icon:n
      { #1 }
    \,
```

File: images/CPU-drawing.pdf
Comment (Witiko) [inline images/CPU-drawing.pdf:1]: The CPU should be thinner than the file icons, not thicker, so that it's clear that the CPU is the background.

File: examples.sty [hunk 2]
```diff
@@ -85,25 +87,31 @@
...
+\cs_set:Nn
+\large_icon:nn
+{
+  \icon:n
+    { #2 }
+       \begin{scope}[every~node/.style={\l_tmpa_tl,fill=white,draw=black,scale=.5},shift={(#1)},scale=.5]
+         \node at (1,-0.85) {};
+         \node at (1.3,-0.55) {};
+         \node at (1.6,-0.15) {};
+         \end{scope}
```
Comment (Witiko) [inline examples.sty:113]: There is a huge gap below the large icon and its label:

### Before
image

### After
image

**Comment by Witiko** [comment]
@TeXhackse Thank you. Can you also please replace `main.tex` and `main.bib` with the files from Karl to this (or another) PR, so that we can review the diff?

File: examples.sty (hunk 2 above)
Comment (TeXhackse) [reply examples.sty:113]: yup i know wanted to know first if you generally are okay wih the icons, will adjust right now

**Comment by TeXhackse** [comment]
> @TeXhackse Thank you. Can you also please replace `main.tex` and `main.bib` with the files from Karl to this (or another) PR, so that we can review the diff?

Sure

File: images/CPU-drawing.pdf
Comment (TeXhackse) [reply images/CPU-drawing.pdf:1]: since i don't want to update hte pdf multiple times in the repo, how about 

image

Comment (TeXhackse) [reply images/CPU-drawing.pdf:1]: I'd also move the file down a bit for the bottom image, to avoid that weird overlap

Comment (TeXhackse) [reply images/CPU-drawing.pdf:1]: image

Comment (Witiko) [reply images/CPU-drawing.pdf:1]: Looks good!

File: examples.sty [hunk 3]
```diff
@@ -244,3 +252,154 @@
...
+    \foreach \y in {-.8,-.4,0} {
+		\pgfpathmoveto{\pgfpointadd{\pgfqpoint{-.65\pgf@xc}{0pt}}{\pgfpoint{0}{\y\pgf@xc}}}%
+		\pgfpathlineto{\pgfpointadd{\pgfpoint{.65\pgf@xc}{0pt}}{\pgfpoint{0}{\y\pgf@xc}}}%
+    }
+   \fi
+  }%
+}%
+
+\tikzset{
+	code-document/.style={document,fill=white,very thick,minimum height=45pt,draw=black,node contents=\ttfamily</>},
```
Comment (Witiko) [inline examples.sty]: Perhaps we can change the font size in `\small_icon:n` and `\large_icon:nn` to make the contents (`</>`) scale better. It's currently quite tiny in the large icon.

Comment (TeXhackse) [reply examples.sty]: in increased it a bit. general issue here is, that we now scale the icon with the font. I added support for reducing the `inner~sep` to the shape. Guess its okay now, but please have a look.

**Comment by Witiko** [comment]
Already fixed in #24.
